src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
+export const addBook = (book) => ({
+  type: ADD, book,
+});
+
+export const removeBook = (book) => ({
+  type: REMOVE, book,
+});
+
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case ADD:
+      return [
+        ...state, action.book,
+      ];
+
+    case REMOVE:
+      return state.filter((book) => book.id !== action.book);
+
```

Comment:
- [ ] I believe here, you want to filter based on the book's id. Your comparison `book.id !== action.book` is incomplete because now you are comparing the a book's id with a book object. Please do 👇 
```
state.filter((book) => book.id !== action.book.id);
```

Reply:
@codepantha thanks for your suggestions. They have been applied now.

Reply:
You're welcome @Alexr16, please go ahead and request another review 👍